### ISSUE TITLE
Does this project have a homepage?

### ISSUE THREAD

**Issue by ohlala** [ossc-db/pg_hint_plan#35]
Does this project have a homepage? I browsed this URL http://pghintplan.osdn.jp/pg_hint_plan.html  but it looks like it is out of date.

**Comment by horiguti** [comment]
Hello.

At Tue, 11 Jun 2019 02:28:12 -0700, KUN <notifications@github.com> wrote in <ossc-db/pg_hint_plan/issues/35@github.com>
> Does this project have a homepage? I browsed this URL http://pghintplan.osdn.jp/pg_hint_plan.html  but it looks like it is out of date.

Yeah, that seems stale..  I'm ashamed but I'm not sure how can I
reach the page.  I would appreciate it if you could tell me how
you reached there.

Even though it's not immediately browsable but you will find the
latest documentation at the following URL:

https://ja.osdn.net/projects/pghintplan/scm/git/pg_hint_plan/tree/master/doc/

regards.

-- 
Kyotaro Horiguchi
NTT Open Source Software Center

**Comment by ohlala** [comment]
![image](https://user-images.githubusercontent.com/13494601/59321117-6034c480-8d03-11e9-9489-251bb8fbc464.png)

**Comment by ohlala** [comment]
I found this link on Google. So I have been using the 1.1 version. . . I ran into some problems and found the latest version on github when I found a solution. Thanks for the tools that have helped me a lot.

**Comment by ohlala** [comment]
I found this link on Google.
![image](https://user-images.githubusercontent.com/13494601/59321431-90309780-8d04-11e9-9425-3863efbb565c.png)
The page looks like this 
![image](https://user-images.githubusercontent.com/13494601/59321668-822f4680-8d05-11e9-8705-ebd8f59d1a82.png)

 I always thought this was the latest version, so I have been using the 1.1 version. . . I discovered the latest version of github until I encountered some problems and sought a solution. 

Thanks for your link, I will change the pghint in my project to the latest one to see if I can solve the problem.

The tools you developed have given me a lot of help, thank you very much.
